t04_refactored.py

```diff
@@ -231,8 +231,36 @@ def team_19_adv():
 ###################################################################################
 
 def team_20_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1aO8daq4WLqOsChlhvmClBHNZEx4faJcmsvGXPyOU990/edit?usp=sharing
+    :return:
+    America Gaona Borges
+    Jessica Douthit
+    """
+    global isDead
+    direction = input("Go with him further Up, or Down? ")
+
+    if direction == "Up":
+        print("You've reached a large vault... a large console stands to its right...")
+        print("The console lights up when you reach it... it asks for a passcode")
+        input("Guess the code" or "Ask for help? ")
+        if "Ask for help? ":
+            print("You ask Dr. Heggen for help")
+            print("He pulls out a small electrical device and plugs it into the console")
+            print("The console glows green and the vault opens up")
+            print("you've found riches beyond your wildest dreams")
+        if isDead:
+            print("You type in '12345' and the console turns red")
+            print("The walls close in on you, crushing you like a Wookie in a trash compactor")
+            isDead = True
+            if isDead:
+                kill_if_dead(isDead)
+
+    else:
+        print("you have reached the strongest creature in all of the lands..")
+        sleep(delay * 3)
+        print("It is the Teacher Assistant Silas! Do you fight or run?")
```

Comment:
Looks like this code ends prematurely. 

Part of the original prompt was to include a "second chance" if the user was about to die (e.g., pick a number between 1 and 10). Looks like that code was supposed to be here, but it didn't get finished. 

You get to finish it!